quic/s2n-quic-core/src/recovery/cubic.rs

```diff
@@ -18,6 +18,10 @@ use core::{
 };
 #[cfg(not(feature = "std"))]
 use num_traits::Float as _;
+use std::env;
```

Comment:
I'm also wondering if we should have the env higher in the stack too. It would be nice to be able to test the new code without having to set env variables.